modules/serverless-channel-default.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+//  * serverless/event_workflows/serverless-channels.adoc
+
+[id="serverless-channel-default_{context}"]
+= Configuring the default channel implementation
+
+The `default-ch-webhook` ConfigMap can be used to specify the default channel implementation for the cluster or for one or more namespaces.
```

Comment:
s/ConfigMap/config map/

Reply:
Instead of putting this same comment on every instance, I'll just say: throughout. :)

Reply:
The updates for the glossary stuff is the next thing on my todo list, definitely need to familiarize myself! Thanks for catching this.